File: Beecrowd/Beginner/1101 - 1190/1149 - Summing Consecutive Integers/C++/1149_SummingConsecutiveIntegers.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int a, numbers, sum = 0;
    cin >> a >> numbers;

    while (numbers <= 0) cin >> numbers;

    for (int i = 0; i < numbers; i++) sum += a + i;

    cout << sum << endl;
    
    return 0;
}

File: Beecrowd/Beginner/1101 - 1190/1149 - Summing Consecutive Integers/Python/1149_SummingConsecutiveIntegers.py
numbers = list(map(int, input().split()))
i = 1
sum = 0

while numbers[i] <= 0:
    if numbers[i] <= 0:
        i += 1
    if numbers[i] > 0:
        break

for j in range(0, numbers[i]):
    sum += numbers[0] + j
    
print(sum)